### ISSUE TITLE
Github Security Alerts (falling behind a bit) 

### ISSUE THREAD

**Issue by tcg** [lifechurch/youversion-web-open-ideas#33]
For admins, Github is displaying the following list of known issues with dependencies of this project. Since this is client-side only, and a "work in progress" project for React components, these aren't *critical* at the moment. That said, it would be nice if things didn't fall too far behind before development ramps up again this year. 

The `yarn audit` list is also *really* long 😬 😬 😬 

Currently, it *might* just be a matter of updating a couple of the base dependencies in our `package.json` file, and making sure things still behave the same afterward. Noting / filing a new issue if something is broken after upgrading dependencies. 

Assumed steps: 

- Get a fresh clone
- Make sure `yarn` is up to date
- Run `yarn` to install dependencies 
- Run `yarn start` and poke around to see how things work currently 
- Review the github list, or `yarn audit`, and update necessary base dependencies
- Run `yarn start` again and check that things behave the same as before 
- - If not, file a new issue that "upgrading dependencies breaks [thing/behavior]" with as much specific detail as might be related (specific dependency updated, specific before/after behavior if possible, etc) 

![image](https://user-images.githubusercontent.com/575773/50857286-c4b5b700-1352-11e9-8538-0622442b3721.png)

**Comment by tcg** [comment]
To be clear, it's not expected that the result of this is eliminating 100% of the output from `yarn audit`. Simply addressing at least the most critical dependencies from the Github provided list (see image in description of this issue). And only if possible -- as it could be that we have to wait on the dependency itself to update, etc.

**Comment by tcg** [comment]
@michael-martin-al  I see we have a `package-lock.json` and a `yarn.lock`. Do you know if we purposely switched some usage to `npm`?  (I suppose I could check git-blame and see when it was introduced).

**Comment by ambersariya** [comment]
Not the most experienced with something like this but happy to give this a go?

**Comment by damcclean** [comment]
I'll have a look into doing this

**Comment by tcg** [comment]
😱 Late to respond, sorry! 

@ambersariya You're totally welcome to fork the repo and give it a shot! Feel free to ask specific questions here if there's something specific you're stuck on / have questions. 

@damcclean Awesome! I'd love for @ambersariya to have a shot at learning/figuring it out, but if you also wanted to ........ oh, I just saw you submitted the PR already! Ha! Awesome. I'm so behind. 😆

**Comment by ambersariya** [comment]
> 😱 Late to respond, sorry!
> 
> @ambersariya You're totally welcome to fork the repo and give it a shot! Feel free to ask specific questions here if there's something specific you're stuck on / have questions.
> 
> @damcclean Awesome! I'd love for @ambersariya to have a shot at learning/figuring it out, but if you also wanted to ........ oh, I just saw you submitted the PR already! Ha! Awesome. I'm so behind. 😆

I'll wait for another issue to pop up :-)
